source/release-notes.html.md

```diff
@@ -22,6 +22,7 @@ Additional changes:
 - Clarify the endpoint for rejecting an application in usage scenarios
 - Clarify the timestamp format for retrieving applications in usage scenarios
 - Add description of the API versioning
+- Add error responses to OpenAPI spec
```

Comment:
Maybe we should be more specific here? We've only added in error responses for the retrieve an application and retrieve many applications endpoints.